packages/eslint-plugin/lib/config/react.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
       '@shopify/react-initialize-state': 'error',
       '@shopify/react-no-multiple-render-methods': 'error',
       '@shopify/react-prefer-private-members': 'error',
+      '@shopify/react-require-autocomplete': 'error',
```

Comment:
I’m guessing you thought this through already but… could there be any case where we wouldn’t want an `autoComplete` on one of these input types? Asking because maybe it should simply be a `warn`.